client/state/activity-log/actions.js

```diff
@@ -163,8 +163,9 @@ export function updateRewindRestoreProgress( siteId, timestamp, restoreId, progr
 	return {
 		type: REWIND_RESTORE_UPDATE_PROGRESS,
 		...progress,
-		siteId,
+		freshness: Date.now(),
```

Comment:
I suppose this could be done in the reducer, meaning no need for this `freshness` field here? Not a big difference, though.